E1_TFHub_Sample_Deploy/export_test.py

```diff
@@ -0,0 +1,55 @@
+import tensorflow as tf
+import numpy as np
+import tensorflow_datasets as tfds
+import tensorflow_hub as hub
+import unittest
+from absl import app, flags
+
+flags.DEFINE_string(
+    "exported_path",
+    "/tmp/tfhub_modules/mnist/digits/1",
+    "Path to exported Module")
+flags.DEFINE_string("data_dir", None, "Path to TFDS Dataset Directory")
+flags.DEFINE_boolean("check_accuracy", False, "Checks the accuracy of a model")
+flags.DEFINE_integer("batch_size", 32, "Batch size for checking accuracy")
+flags.DEFINE_integer(
+    "buffer_size",
+    1000,
+    "Buffer size used for Data Shuffling")
+FLAGS = flags.FLAGS
```

Comment:
Your unit tests shouldn't contain command line flags, since this isn't a script really. Once you refactor `export.py` to have functions that take these as arguments, you will need to implement tests that call those functions with appropriate arguments.

See how `export_path` is passed as an argument [here](https://github.com/tensorflow/hub/blob/master/examples/text_embeddings/export_test.py#L67)

Reply:
Done :)